lib/dl_api_client/pyproject.toml

```diff
@@ -42,6 +42,10 @@ disallow_untyped_defs = true
 check_untyped_defs = true
 strict_optional = true
 
+[[tool.mypy.overrides]]
+module = "marshmallow_oneofschema.*"
+ignore_missing_imports = true
+
```

Comment:
Didn't you mention that the new version has become better friends with mypy?

Reply:
You are right. I've forgotten to remove this after rebase.